src/hooks/hardhead/useAwardWinners.ts

```
@@ -0,0 +1,69 @@
+import { useState, useEffect } from "react";
+import config from "react-global-configuration";
+import axios, { AxiosError } from "axios";
+import { WinnerEntity } from "../../types/winnerEntity";
+
+import { useState, useEffect } from "react";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useState'. Consider to delete it or rename it.

'useState' is defined here:


(lint/suspicious/noRedeclare)